src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1032,32 +1032,39 @@ void setCandidateExclusionStatus(String candidateCode, boolean excluded) {
   }
 
   // perform pre-processing on candidates:
-  // 1) build map of candidate ID to candidate name
+  // 1) build map of candidate aliases to candidate name
   // 2) generate tie-break ordering if needed
   // 3) add uwi candidate if needed
   private void processCandidateData() {
-    candidateCodeToNameMap = new HashMap<>();
+    candidateAliasesToNameMap = new HashMap<>();
+    candidateAliasesToCodeMap = new HashMap<>();
+    candidateNames = new HashSet<>();
```

Comment:
similarly, why not just initialize these when you declare them?

Reply:
Just following the existing model, a la `validateCandidates`. I don't mind changing these to init when declared, but perhaps we should have a larger movement to do that consistently across this and other files?